src/read.c

```diff
@@ -378,92 +380,92 @@ static void AssignRef(const LHSRef ref)
     }
 }
 
-static void UnbindRef(ScannerState * s, const LHSRef ref)
+static void UnbindRef(ReaderState * rs, const LHSRef ref)
```

Comment:
Same as `EvalRef`; note that the `ScannerState` is contained in the `ReaderState`, so there is no need for us to pass both.